src/Roslyn.Diagnostics.Analyzers/Core/SymbolIsBannedAnalyzer.cs

```diff
@@ -245,7 +245,7 @@ void VerifyDocumentationSyntax(Action<Diagnostic> reportDiagnostic, SyntaxNode s
                 {
                     VerifyType(reportDiagnostic, typeSymbol, syntaxNode);
                 }
-                else
+                else if (symbol != null)
```

Comment:
Do we have a unit test/repro?

Reply:
I saw a stack trace in the IDE but haven't been able to trigger it again since. I'll try and get a repro, then add a test case.

Reply:
I've looked but haven't found a repro for this. The documentation states the return value can be null, so this would've been picked up with nullable ref types. Can we call this a type error and never talk about it again? 😉

Reply:
I am completely fine, was just trying to see if it was easy to get a repro. Feel free to merge.